src/app/routes/index.js

```diff
@@ -22,14 +23,14 @@ const routes = [
   {
     path: articleRegexPath,
     exact: true,
-    component: Article,
+    component: withVariant(Article),
```

Comment:
Can this instead be added [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Article/index.jsx#L26) please? Its designed to have them stack

Reply:
Fixed!